index.html

```diff
@@ -125,6 +129,297 @@ <h1>I'm Moise Glad to see you !</h1>
 					</li>
 				</ul>
 			</section>
+			<section class="works">
+				<div class="card-work">
+					<a class="portfolio-img" href="#">
+						<img src="/images/Portfolio_1.png" alt="tonic" />
```

Comment:
- [ ] **The project images are not loading on your website. Please fix it.**

![image](https://user-images.githubusercontent.com/59168430/120854595-02a67c00-c59b-11eb-8e67-97dece706744.png)

The images are probably not showing up because of the bad path. Following is how HTML understands the paths. Please compare them and try to determine which one will be the most suitable in your case. 

`/` means the root of the current drive or web;

`./` means the current directory;

`../` means the parent of the current directory.

Reply:
This issue is addressed! :clap: